src/dev/performance/run_scalability_cli.ts

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { run } from '@kbn/dev-cli-runner';
+import { REPO_ROOT } from '@kbn/utils';
+import fs from 'fs';
+import path from 'path';
+
+run(
+  async ({ log, flagsReader, procRunner }) => {
+    async function runScalabilityJourney(journeyConfigPath: string) {
+      // Pass in a clean APM environment, so that FTR can later
+      // set it's own values.
+      const cleanApmEnv = {
+        ELASTIC_APM_ACTIVE: undefined,
+        ELASTIC_APM_BREAKDOWN_METRICS: undefined,
+        ELASTIC_APM_CONTEXT_PROPAGATION_ONLY: undefined,
+        ELASTIC_APM_CAPTURE_SPAN_STACK_TRACES: undefined,
+        ELASTIC_APM_ENVIRONMENT: undefined,
+        ELASTIC_APM_GLOBAL_LABELS: undefined,
+        ELASTIC_APM_MAX_QUEUE_SIZE: undefined,
+        ELASTIC_APM_METRICS_INTERVAL: undefined,
+        ELASTIC_APM_SERVER_URL: undefined,
+        ELASTIC_APM_SECRET_TOKEN: undefined,
+        ELASTIC_APM_TRANSACTION_SAMPLE_RATE: undefined,
+      };
+
+      await procRunner.run('scalability-tests', {
+        cmd: 'node',
+        args: [
+          'scripts/functional_tests',
+          ['--config', 'x-pack/test/scalability/config.ts'],
+          ['--kibana-install-dir', kibanaInstallDir],
+          '--debug',
+          '--logToFile',
+          '--bail',
+        ].flat(),
+        cwd: REPO_ROOT,
+        wait: true,
+        env: {
+          ...cleanApmEnv,
```

Comment:
Maybe we should add an option to `procRunner` which prevents extending the current ENV, something like `extendEnv: false`

Reply:
Good idea @spalger , it is still a draft PR, maybe I accidentally moved it from draft state.